src/com/xilinx/rapidwright/rwroute/RWRoute.java

```diff
@@ -122,6 +121,11 @@ public class RWRoute{
     /** Flag for whether LUT routethrus are to be considered */
     protected boolean lutRoutethru;
 
+    /** Flag for use of Hybrid Updating Strategy (HUS) */
+    private boolean useHUS;
+    /** Flag (computed at end of iteration 1) to indicate design is congested enough to consider HUS */
+    private boolean HUSinitialCongested;
```

Comment:
Could we refactor `HUSinitialCongested` to not begin with a capital letter since it is not a class or enum type?  Typical Java convention has classes and enums starting with a capital letter with variables starting with a lowercase letter (see lowerCamelCase in https://github.com/Xilinx/RapidWright/blob/master/CODING_STYLE.md).